manual/source/why.rst

```diff
@@ -0,0 +1,166 @@
+Why does this exist?
+~~~~~~~~~~~~~~~~~~~~
+
+Central to Rotonda is the observation that there is no one-size-fits-all
+application for a network's BGP routing needs any more. As networks are
+growing, so is their complexity, and as a consequence, BGP is being used in
+more places in the network, and it has turned into a generic container for
+information about network nodes, increasing its complexity. Pressure on BGP to
+offer better security further increased the complexity of BGP itself and its
+deployments.
+
+By now, the notion of a 'BGP speaker' (the wording of RFC4271) with a fixed
+set of 'Routing Information Bases' (RIBs) with prescribed behaviour is only
+one of many different "BGP functions", as we would like to call them, that we
+can identify in a network. Rotonda aims to provide users with the building
+blocks to create their own application optimized for any of these BGP
+functions.
+
+BGP Functions
+~~~~~~~~~~~~~
+
+So, what are these BGP functions anyway? The way we would like to describe them
+is "a specialised task within a network relying on BGP, that requires a
+subset of BGP's features". Often, this task is performed by a node in the
+network.
+
+BGP's Features
+~~~~~~~~~~~~~~
+
+Before we delve deeper into BGP functions, let's first establish what we mean
+with BGP's features, by posing the question: "What are the
+requirements for a fully-fledged BGP speaker?". RFC4271 and its updates
+describes BGP consisting of three components: the BGP packet format, the BGP
+state machine, and the Routing Information Base.
+
+The specific required interaction of these components describe BGP's features,
+they boil down to:
+
+"Listening", a.k.a "passively speaking" BGP
+    - parsing BGP packets
+    - opening and maintaining sessions with the BGP state machine
+
+    This is the least amount of work a BGP function would need to do in order
+    to be accepted as a functioning part of a network routed with BGP.
+
+Keeping State in RIBs called
+    - `adj-RIB-in-pre/post`,
+    - `loc-RIB`
+    - `adj-RIB-out`
```

Comment:
Do you intend this to be italic or monospace red text? If you wanted the latter, you need a double backtick, i.e. ``adj-RIB-out``

Reply:
If it's up to me I would say italic